### PR TITLE
Feat/prep 2020.10

### PR DESCRIPTION
**Pull request by aodukha**
preparing 2020.10 release

### CODE REVIEW

File: CHANGELOG.md
Comment (patrickvacek) [inline CHANGELOG.md:15]: I think it's worth specifying which dockerfile, so I'd just copy the PR title:
```suggestion
- Use default ostree package in Ubuntu Focal dockerfile: [PR](https://github.com/advancedtelematic/aktualizr/pull/1751)
```

File: CHANGELOG.md
Comment (patrickvacek) [inline CHANGELOG.md:21]: I would put the two items that start with "Fixed" into a separate *Fixed* subsection like in previous releases.

**Review by patrickvacek — COMMENTED** [review]
Cool, looks good! Just a couple minor things from my side.

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
- Ubuntu Focal Dockerfile now uses the default OSTree package: [PR](https://github.com/advancedtelematic/aktualizr/pull/1751)
```

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md:16]:
```suggestion
- Improved libaktualizr API exceptions: [PR](https://github.com/advancedtelematic/aktualizr/pull/1754)
```

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md:17]:
```suggestion
- Improved binary file download progress: [PR](https://github.com/advancedtelematic/aktualizr/pull/1756)
```

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
- Fixed the issue with the parameters check in `aktualizr-get`: [PR](https://github.com/advancedtelematic/aktualizr/pull/1760)
```

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md:18]:
```suggestion
- Allowed passing HTTP headers in `aktualizr-get`: [PR](https://github.com/advancedtelematic/aktualizr/pull/1762)
```

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md:19]:
```suggestion
- Moved aktualizr-lite to its own [aktualizr-lite repository](https://github.com/foundriesio/aktualizr-lite): [PR](https://github.com/advancedtelematic/aktualizr/pull/1763)
```

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md:23]:
```suggestion
- Fixed the output of the pacman configuration: [PR](https://github.com/advancedtelematic/aktualizr/pull/1761)
```

**Review by HalynaDumych — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md:12]:
```suggestion
- Updated the `garage-push` and `garage-deploy` tools. Now, they support new back-end token generation to authenticate API requests. Also, we updated the `treehub.json` format for the new back-end. It now has the additional  *scope* parameter. The changes are backward compatible. Previous versions have the server URL *without* the token path, so it needs to be hardcoded. The new version has the full URL with the */oauth2/token* path at the end: [PR](https://github.com/advancedtelematic/aktualizr/pull/1767)
```

**Comment by patrickvacek** [comment]
Not critical, but it appears all of the commits have been flattened into one. When porting commits from one branch to another, I think it's best to keep them as close to the original as much as possible. I think it shouldn't really matter in this case, so I'm not going to make a requirement before merging this PR.

File: CHANGELOG.md
Comment (HalynaDumych) [inline CHANGELOG.md:12]:
```suggestion
- Updated the `garage-push` and `garage-deploy` tools. Now, they support new back-end token generation to authenticate API requests. Also, we updated the `treehub.json` format for the new back-end. It now has the additional  `scope` parameter. The changes are backward compatible. Previous versions have the server URL **without** the token path, so it needs to be hardcoded. The new version has the full URL with the `/oauth2/token` path at the end: [PR](https://github.com/advancedtelematic/aktualizr/pull/1767)
```

**Review by HalynaDumych — DISMISSED** [review]
Just one minor comment.
The rest looks good.

**Review by HalynaDumych — APPROVED** [review]

**Review by aodukha — COMMENTED** [review]
moved into Fixed section

**Review by HalynaDumych — APPROVED** [review]

**Review by patrickvacek — APPROVED** [review]
LGTM, thanks! Since this is docs/text-only we don't have to wait for CI.

**Comment by aodukha** [comment]
> Not critical, but it appears all of the commits have been flattened into one. When porting commits from one branch to another, I think it's best to keep them as close to the original as much as possible. I think it shouldn't really matter in this case, so I'm not going to make a requirement before merging this PR.

yep. at least there are messages from these commits in the comment so it is still possible understand what happens

**Pull request merged**
